backend/schema/room.py

```diff
@@ -30,6 +33,7 @@ class Plugin(str, Enum):
     """
 
     DM = "DM"
+    GROUP_DM = "GROUP_DM"
     CHANNEL = "Channel"
```

Comment:
Can we?
```suggestion
    CHANNEL = "CHANNEL"
```
As with DM & GROUP_DM

Reply:
fixed